api/ask_astro/config.py

```diff
@@ -61,3 +61,13 @@ class WeaviateConfig:
     index_name = os.environ.get("WEAVIATE_INDEX_NAME")
     text_key = os.environ.get("WEAVIATE_TEXT_KEY")
     attributes = os.environ.get("WEAVIATE_ATTRIBUTES", "").split(",")
+
+
+class MetricsDBConfig:
+    """Containers the config variables for Metrics DB"""
+
+    user = os.environ.get("METRICS_DB_USER")
```

Comment:
should we name it `METRICS_SNOWFLAKE_DB_USER`? and change others as well?

Reply:
We could. Just rename it. Thanks!